packages/storybook/stories/Examples/TeamsInterop/Lobby.stories.tsx

```diff
@@ -10,10 +10,13 @@ import { Lobby as LobbyComponent } from './snippets/Lobby.snippet';
 import { getDocs } from './TeamsInteropDocs';
 
 const LobbyStory: () => JSX.Element = () => {
-  const callStateText = text('Call State Text', 'Waiting for others to join');
+  const callStateText = text('Call State Text', "You're in the lobby");
+  const callStateSubText = text('Call State Subtext', 'You should be admitted shortly');
```

Comment:
Is this text inline with the Figma?
@pereiralex 

Could you point us to the figma you used for this @JoshuaLai 
Given that the user can be kicked out while in lobby, it seems a little misleading.

Reply:
@anjulgarg figma link here. 
https://www.figma.com/file/72qJ03EOQExf12g5JTdWjQ/ACS-UI?node-id=1188%3A0

Reply:
Thanks. Resolving this.